### PR TITLE
Fixed support for ViaFabricPlus 3.1.0 and newer versions

### PR DESCRIPTION
**Pull request by florianreuth**
Added support for the new VFP version / the new API changes from VV, once 1.20.5 releases, the old code can be deleted.

### CODE REVIEW

File: src/main/java/net/earthcomputer/clientcommands/MultiVersionCompat.java [hunk 1]
```diff
@@ -40,10 +43,11 @@ public boolean doesItemExist(Item item) {
     public static final MultiVersionCompat INSTANCE = Util.make(() -> {
         try {
             FabricLoader loader = FabricLoader.getInstance();
-            if (loader.isModLoaded("viafabric")) {
+            ModContainer modContainer;
+            if (loader.getModContainer("viafabric").orElse(null) != null) {
```
Comment (crosby-moe) [inline src/main/java/net/earthcomputer/clientcommands/MultiVersionCompat.java]: why use `loader.getModContainer("viafabric").orElse(null) != null` if you dont capture the `ModContainer`?

Comment (florianreuth) [reply src/main/java/net/earthcomputer/clientcommands/MultiVersionCompat.java]: oh yeah right, I can probably just use isModLoaded() here

Comment (florianreuth) [reply src/main/java/net/earthcomputer/clientcommands/MultiVersionCompat.java]: Fixed

**Comment by florianreuth** [comment]
Applied Earth's suggestion from dc and splitted VFP support into two class files

**Pull request merged**